Console.java

```diff
@@ -0,0 +1,191 @@
+                    reader.getTerminal().puts(InfoCmp.Capability.clear_screen);
+                } else if (command instanceof TransactionReplCommand.Help) {
+                    printer.info(TRANSACTION_HELP_MENU);
+                } else if (command instanceof TransactionReplCommand.Commit) {
+                    tx.commit();
+                    printer.info("Transaction changes committed");
+                    break;
+                } else if (command instanceof TransactionReplCommand.Rollback) {
+                    tx.rollback();
+                    printer.info("Rolled back to the beginning of the transaction");
+                } else if (command instanceof TransactionReplCommand.Close) {
+                    tx.close();
+                    printer.info("Transaction closed without committing changes");
+                    break;
+                } else if (command instanceof TransactionReplCommand.Source) {
+                    String queryString;
+                    try {
+                        queryString = new String(Files.readAllBytes(Paths.get(command.asSource().file())), StandardCharsets.UTF_8);
+                    } catch (IOException e) {
+                        printer.error("Failed to open file '" + command.asSource().file() + "'");
+                        continue;
+                    }
+                    runQuery(tx, queryString, printer);
+                } else if (command instanceof TransactionReplCommand.Query) {
+                    runQuery(tx, command.asQuery().query(), printer);
+                }
+            }
+        } catch (GraknClientException e) {
+            printer.error(e.getMessage());
+        }
+        return false;
+    }
+
+    private static void runQuery(Grakn.Transaction tx, String queryString, Printer printer) {
+        List<GraqlQuery> queries;
+        try {
+            queries = Graql.parseQueries(queryString).collect(Collectors.toList());
+        } catch (GraqlException e) {
+            printer.error(e.getMessage());
+            return;
+        }
+        for (GraqlQuery query : queries) {
+            if (query instanceof GraqlDefine) {
+                tx.query().define(query.asDefine()).get();
+                printer.info("Concepts have been defined");
+            } else if (query instanceof GraqlUndefine) {
+                tx.query().undefine(query.asUndefine()).get();
+                printer.info("Concepts have been undefined");
+            } else if (query instanceof GraqlInsert) {
+                Stream<ConceptMap> result = tx.query().insert(query.asInsert()).get();
+                result.forEach(cm -> printer.info(cm + " has been inserted"));
+            } else if (query instanceof GraqlDelete) {
+                tx.query().delete(query.asDelete()).get();
+                printer.info("Concepts have been deleted");
+            } else if (query instanceof GraqlMatch) {
+                throw new GraknClientException("Match query is not yet supported");
```

Comment:
Client doesn't seem to throw exceptions currently, we simulate the exceptions for now.